src/assets/content/dictionary/bitmap.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: bitmap
-definition: ''
-perspectives: []
+definition: 'an image made up with an arrangement of dots, displayed on the screen. The dots are represented by rgb (red, green, blue) pixels. Best viewed at 100% or lower resolution'
+perspectives: [bitmap images have a finite resolution, above 100%; the image is pixelated and no longer clear to view, looks blurred]
```

Comment:
```suggestion
definition: An image made up with an arrangement of dots, displayed on the screen. The dots are represented by rgb (red, green, blue) pixels. Best viewed at 100% or lower resolution. Bitmap images have a finite resolution, above 100%; the image is pixelated and no longer clear to view, looks blurred.
```
Bug: Since these strings are both parts of the definition, we need to have them in one line.

Reply:
hi @novellac thank you for the message; I agree with the changes made and this PR to be merged
What would be the best way to describe the 'perspective' ?

Reply:
Hi @goobric thanks for asking about writing a perspective. Many people reading our dictionary may be new to tech, and even thought they may understand a definition, they might not know how the word is used by different people on a team, or the significance it might hold for them.
The [example in the contribution docs](https://github.com/cherryontech/website/blob/pit/CONTRIBUTING.md#tech-dictionary-example) isn't a tech term, but I think it's a good one - a "nap" means something very different from the perspective of a tired caretaker or an energetic child!